Documentation/bash-scripts.md

```diff
@@ -0,0 +1,52 @@
+# Bash scripts
+
+A common way to use acbuild is via a bash script. There's a common template
```

Comment:
Bash scripting provides a common, simple, and well-known mechanism to drive `acbuild`s. A script template, below, is used for the examples here.